universe/Effect.h

```diff
@@ -79,6 +80,7 @@ class FO_COMMON_API EffectsGroup {
     std::string                 m_accounting_label;
     int                         m_priority;
     std::string                 m_description;
+    std::string	                m_content_name;
```

Comment:
tab

Reply:
Just do a find and replace... don't care if an intermediate commit has a tab if the final pull request changes are clean.

Reply:
> Just do a find and replace

Ah, OK, I just found the control to let me use regular expressions so I could enter a tab into the find box.   Using left and right arrows to navigate across that line they had all seemed like spaces. Still learning my way around VS, sorry...

Reply:
I think you can just copy paste a tab character into the find box as well.